package.json

```diff
@@ -152,7 +151,7 @@
 		"check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2\" \"wp-scripts check-licenses --dev\"",
 		"precheck-local-changes": "npm run docs:build",
 		"check-local-changes": "( git diff -U0 | xargs -0 node bin/process-git-diff ) || ( echo \"There are local uncommitted changes after one or both of 'npm install' or 'npm run docs:build'!\" && exit 1 );",
-		"ci": "concurrently \"npm run lint\" \"npm run test-unit:coverage-ci\" \"npm run check-local-changes\"",
+		"ci": "concurrently \"npm run lint\" \"npm run check-local-changes\"",
```

Comment:
`test-unit:coverage-ci` should still be here we just need to update it as follows:
```json
"test-unit:coverage-ci": "npm run test-unit -- --ci --runInBand",
```